docs/hub/enterprise-hub-gating-group-collections.md

```diff
@@ -0,0 +1,166 @@
+        alt="Hugging Face gating group collection settings with gating disabled"
+    />
+</div>
+
+By default, access to the repos in the collection is automatically granted to the user when requesting it. This is referred to as **automatic approval**. In this mode, any user can access your repos once they’ve shared their contact information with you.
+
+<div class="flex justify-center" style="max-width: 750px">
+    <img
+        class="block dark:hidden !m-0"
+        src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/collections/gating-group-modal-enabling.webp"
+        alt="Hugging Face gating group collection settings with automatic mode selected"
+    />
+    <img
+        class="hidden dark:block !m-0"
+        src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/collections/dark-gating-group-modal-enabling.webp"
+        alt="Hugging Face gating group collection settings with automatic mode selected"  
+    />
+</div>
+
+If you want to manually approve which users can access repos in your collection, you must set it to **Manual Review**. When this is the case, you will notice a new option:
+
+**Notification frequency** lets you configure when to get notified about new users requesting access. It can be set to once a day or real-time. By default, emails are sent to the first 5 admins of the organization. You can also set a different email address in the **Notifications email** field.
+
+<div class="flex justify-center" style="max-width: 750px">
+    <img
+        class="block dark:hidden !m-0"
+        src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/collections/gating-group-modal-manual.webp"
+        alt="Hugging Face gating group collection settings with manual review mode selected"
+    />
+    <img
+        class="hidden dark:block !m-0"
+        src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/collections/dark-gating-group-modal-manual.webp"
+        alt="Hugging Face gating group collection settings with manual review mode selected"
+    />
+</div>
+
+### Review access requests
+
+Once access requests are enabled, you have full control of who can access repos in your gating group collection, whether the approval mode is manual or automatic. You can review and manage requests either from the UI or via the API.
+
+**Approving a request for a repo in a gating group collection will automatically approve access to all repos in that collection.**
+
+#### From the UI
+
+You can review who has access to the repos in your Gating Group Collection from the settings page of any of the repos of the collection by clicking on the **Review access requests** button:
+
+<div class="flex justify-center" style="max-width: 750px">
+    <img
+        class="block dark:hidden !m-0"
+        src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/collections/gating-group-repo-settings.webp"
```

Comment:
If possible, I'd suggest to change the name of the collection to something like "Awesome Models" instead of "With duplicate repos"

Reply:
Definitely a mistake, will change it 👍